docs/iris/src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,193 @@
+  When an Iris climatological coordinate is saved in NetCDF, the NetCDF time
+  axis will be given a 'climatology' attribute, and the contents of the
+  coordinate's ``bounds`` property are written to a NetCDF boundary variable
+  called 'climatology_bounds'.  These are in place of the more usual NetCDF
+  'bounds' attribute and accompanying 'bnds' boundary variable. See below
+  for an example:
+
+    .. code-block::
+
+        dimensions:
+          time=4;
+          nv=2;
+        variables:
+          float temperature(time,lat,lon);
+            temperature:long_name="surface air temperature";
+            temperature:cell_methods="time: minimum within years time: mean over years";
+            temperature:units="K";
+          double time(time);
+            time:climatology="climatology_bounds";
+            time:units="days since 1960-1-1";
+          double climatology_bounds(time,nv);
+        data:  // time coordinates translated to date/time format
+          time="1960-4-16", "1960-7-16", "1960-10-16", "1961-1-16" ;
+          climatology_bounds="1960-3-1",  "1990-6-1",
+                             "1960-6-1",  "1990-9-1",
+                             "1960-9-1",  "1990-12-1",
+                             "1960-12-1", "1991-3-1" ;
+
+  If a climatological time axis is detected when loading NetCDF -
+  indicated by the format described above - the ``climatological`` property
+  of the Iris coordinate will be set to ``True``.
+
+.. admonition:: New Chunking Strategy
+
+  Iris now makes better choices of Dask chunk sizes when loading from NetCDF
+  files: If a file variable has specified, small chunks, Iris will now choose
+  Dask chunks which are a multiple of these up to a default target size.
+
+  This is particularly relevant to files with an unlimited dimension, which
+  previously could produce a large number of small chunks. This had an adverse
+  effect on performance.
+
+  In addition, Iris now takes its default chunksize from the default configured
+  in Dask itself, i.e. ``dask.config.get('array.chunk-size')``.
+
+.. admonition:: Lazy Statistics
+
+  Several statistical operations can now be done lazily, taking advantage of the
+  performance improvements offered by Dask:
+
+  * :meth:`~iris.cube.Cube.aggregated_by`
+  * :class:`~iris.analysis.RMS` (more detail below)
+  * :class:`~iris.analysis.MEAN`
+
+|
+
+* Cube data equality testing (and hence cube equality) now uses a more relaxed
+  tolerance : This means that some cubes may now test 'equal' that previously
+  did not.
+  Previously, Iris compared cube data arrays using "abs(a - b) < 1.e-8".
+  We now apply the default operation of :func:`numpy.allclose` instead,
+  which is equivalent to "abs(a - b) < (1.e-8 + 1.e-5 * b)".
+* Added support to render HTML for :class:`iris.cube.CubeList`s in Jupyter Notebooks and JupyterLab.
+* Loading CellMeasures with non-float values is now supported.
+* New coordinate system: :class:`iris.coord_systems.Geostationary`, based on
```

Comment:
Do we also load+save these in netcdf ?  If so, should say so (like the one below).

Reply:
👍